rebar.config

```diff
@@ -13,4 +13,8 @@
 
 {shell, [
     {apps, [sscg_generator]}
-]}.
\ No newline at end of file
+]}.
+
+{profiles, [
+    {test, [{deps, [meck]}]}
```

Comment:
Could you declare this dependency outside of a profile?
Two reasons why I don't really like this:
- Building for "usage" (`rebar3 escriptize`) won't pull in all dependencies. I'll have to recompile with the `test` profile if I don't wan't my code editor to complain about unresolved references in the test suite file.
- Dependencies outside of the root `deps` section are not locked in the `rebar.lock` file.

It's not a big deal at all that we'll download `meck` when we run `rebar3 escriptize` this way.
And btw, this is even less of a problem when we build releases. Deps will only be included in the release if they are present in the app.src file or the release specification. So we can just dump every dependency into the main `deps` section without bloating the release artifact with unused dependencies.